libs/logging/package.json

```diff
@@ -22,8 +22,8 @@
     "lint": "pnpm type-check && eslint .",
     "lint:fix": "pnpm type-check && eslint . --fix",
     "test": "is-ci test:ci test:watch",
-    "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage",
+    "test:watch": "TZ=UTC jest --watch",
+    "test:coverage": "TZ=UTC jest --coverage",
```

Comment:
I think we should use `Alaska/Anchorage` instead of `UTC` for consistency with the rest of VxSuite

Reply:
https://github.com/votingworks/vxsuite/issues/5733